runtime/src/assets/stylesheets/shared/main.less

```diff
@@ -57,20 +57,16 @@ body {
 .dn { display: none; }
 .dib { display: inline-block; }
 .db { display: block; }
-.textCenter { text-align: $textCenter; }
+.textCenter { text-align: center; }
```

Comment:
There are most of the classes which follow camelcase. I would prefer to change all of the classes in next PR